core/src/test/resources/oracle-schema-overrides.conf

```diff
@@ -0,0 +1,72 @@
+# Oracle does not support returning a column with a case senstive name so all the column names and table names need
+# to be caps. See:
+# https://github.com/slick/slick/issues/47
+# https://groups.google.com/g/scalaquery/c/U431n-Z2cwM
```

Comment:
Oh, is that the reason why we are still supporting custom column names? How was this working before. 

I also find this strange, I don't think this is an issue in akka projections. At least not for oracle.

Reply:
It is only an issue when you have a prepared statement that returns a value (in our case the ordering id), this isn't supported for case sensitive names in the oracle jdbc driver